gpt_json/tests/test_streaming.py

```diff
@@ -0,0 +1,268 @@
+import json
+from typing import Any, List
+from unittest.mock import MagicMock, patch
+
+import openai
+import pytest
+import tiktoken
+from pydantic import BaseModel
+
+from gpt_json.gpt import GPTJSON
+from gpt_json.models import GPTMessage, GPTMessageRole, GPTModelVersion
+from gpt_json.streaming import StreamEventEnum, StreamingObject
+
+enc = tiktoken.get_encoding("cl100k_base")
+
+def _tokenize(text):
```

Comment:
Consider refactoring these into a separate file, like `streaming_helpers.py` to keep a clean line between what streaming code you're testing and what helps you test it. I see you also have a few unit tests for these functions by themselves (which is a great idea) - you could put these into something like `test_streaming_helpers.py` to keep the logic/test separation of concerns.